pkg/operator/operator_test.go

```diff
@@ -487,6 +487,12 @@ spec:
   helmUpgradeFlags:
     - --set
     - extraValue=repl{{ ConfigOption "deploy_extra_value" }}
+  optionalValues:
+  - when: "repl{{ HasLocalRegistry }}"
+    recursiveMerge: true
+    values:
+      global:
+        registry: ''
```

Comment:
Without these changes this test will fail with the following error since `optionalValues` would also be rendered and the result would not be a string since quotes are stripped.
```
failed to load helm charts: failed to decode kots app data of length 472: json: cannot unmarshal bool into Go struct field OptionalValue.items.spec.optionalValues.when of type string
```